core/arch/x86/proc.c

```diff
@@ -355,8 +357,14 @@ proc_init_arch(void)
                       (!proc_has_feature(FEATURE_FXSR) && !proc_has_feature(FEATURE_SSE)),
                   "Unsupported processor type: SSE and FXSR must match");
 
+    /* XXX i#1312: this will be set based on processor and OS feature support. */
```

Comment:
But isn't it already set?  Please spell out what remains to be done (i.e., why is there a XXX).

Reply:
See above.